apollo-environment/src/main/java/com/spotify/apollo/environment/ApolloEnvironmentModule.java

```diff
@@ -65,8 +65,14 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(ApolloEnvironmentModule.class);
 
-  public static ApolloEnvironmentModule create() {
```

Comment:
Yes, this is an API-breaking change. Not in a big way, but yes.

Not specifying an order can be done like so with the current API:
```
  ApolloEnvironmentModule.create((l, r) -> 0);
```

Not a big deal, but I kind of think it's nice to not pollute the API with two constructor methods - I don't know that we have a use case where we don't care about the decorator ordering? At least, we care about it in the currently known ones (HTTP service, Hermes service, and ServiceHelper).